app/src/colorpalettewidget.cpp

```diff
@@ -143,6 +151,45 @@ void ColorPaletteWidget::removeItem()
     clickRemoveColorButton();
 }
 
+void ColorPaletteWidget::replaceLineColorSingle()
```

Comment:
replaceLineColor is singular already, there's no need to add "Single" to it, it would make more sense to me if the method names were switched. This should be named replaceLineColor and the method below should be called replaceLineColorForAllKeyframes.

Reply:
Sounds reasonable.
I made 'replaceLineColor' first. 
When I later came to think of the possible wish to change line colors for only one drawing, I just added 'Single' to the function name. Your suggestions are clearer.